migrations/002.do.characters-update.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE characters ADD CONSTRAINT characters_unique_name UNIQUE (player_id, name);
```

Comment:
Noticed we were allowing a player to create 3 characters named Robert... which is confusing I think. This constraint limits it so that each player can have only one character with the same name. Different players can still have characters with the same name though.